mailbox/postgres/src/main/java/org/apache/james/mailbox/postgres/mail/PostgresHighestModSeqModule.java

```diff
@@ -0,0 +1,52 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.postgres.mail;
+
+import static org.jooq.impl.SQLDataType.BIGINT;
+
+import java.util.UUID;
+
+import org.apache.james.backends.postgres.PostgresModule;
+import org.apache.james.backends.postgres.PostgresTable;
+import org.jooq.Field;
+import org.jooq.Record;
+import org.jooq.Table;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+
+public interface PostgresHighestModSeqModule {
+    interface PostgresHighestModSeqTable {
```

Comment:
Because `MailboxSessionMapperFactory.getUidProvider()`
was not resolved by MailboxSession parameter, 
-> 
The new separate table will help us don't need concern the row level security of the Mailbox table

Reply:
After looking at it for a while, I agree.

In the core interface we do not resolve it with session param, it's better to create separate tables without RLS to treat those than refactoring the code base for this

Reply:
Me I do not understand why we can not use the Mailbox table directly here...